src/stores/api/handlers/stores.go

```diff
@@ -46,8 +51,42 @@ func (h *StoresHandler) Register(router *mux.Router) {
 	h.eth.Register(ethSubrouter)
 }
 
-func StoreSelector(h http.Handler) http.Handler {
+func storeSelector(h http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		h.ServeHTTP(w, r.WithContext(WithStoreName(r.Context(), mux.Vars(r)["storeName"])))
 	})
 }
+
+func getLimitOffset(request *http.Request) (int, int, error) {
```

Comment:
something is weird with the logic of this function. Shouldn't the logic be:
- `limit` and `offset` are DEFAULT_PAGE_SIZE, 0 by default (named vars from signature)
- If `rLimit` and `rPage` are empty: return DEFAULT_PAGE_SIZE, 0, nil
- If `rLimit` is not empty: limit = transform to uint (no negative values) rLimit
- If `rPage` is not empty: offset = limit * transform to uint(rOffset)
- return limit, offset

Basically reversing the logic.

Reply:
better name the returned values since the function returns 3

Reply:
> limit and offset are DEFAULT_PAGE_SIZE, 0 by default (named vars from signature)
Nop

> If rLimit and rPage are empty: return DEFAULT_PAGE_SIZE, 0, nil
I don't want to force any limit size without user's to now, so that is why I implemented in a way there is not limit size UNLESS user set a page value

>If rLimit is not empty: limit = transform to uint (no negative values) rLimit
good point

>If rPage is not empty: offset = limit * transform to uint(rOffset)
Done

> return limit, offset
I still need to check for errors on values